packages/jest-util/src/isPromise.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// capture global.Promise before it may potentially be overwritten
+const Promise: any = global.Promise;
```

Comment:
_If_ needed, why type it as `any`?

Reply:
`global.Promise` is `Function`, so it complains about `Function` not having a `resolve` property